tools/pot/openvino/tools/pot/graph/passes.py

```diff
@@ -150,7 +150,11 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
         if m_op.type in ['Convolution', 'ConvolutionBackpropData', 'MatMul']:
             insert_fake_quantize(graph, m_op, [0, 1], hw_config=self.hardware_config, input_priority_types=self.input_priority_types)
         elif m_op.type == 'LSTMCell':
-            insert_fake_quantize(graph, m_op, [0, 1, 2, 3, 4], hw_config=self.hardware_config, input_priority_types=self.input_priority_types)
+            insert_fake_quantize(graph, m_op, [0, 1, 3, 4], hw_config=self.hardware_config, input_priority_types=self.input_priority_types)
```

Comment:
Maybe we should take these parameters to another variable and place it somewhere like `special_operations.py`.
For example: 
`types_to_quantizable_ports = {"LSTMCell": [0, 1, 3, 4], ... }`